packages/components/src/box-control/index.js

```diff
@@ -55,7 +55,10 @@ export default function BoxControl( {
 	splitOnAxis = false,
 	allowReset = true,
 	resetValues = DEFAULT_VALUES,
+	spacingType,
 } ) {
+	inputProps = spacingType === 'Margin' ? -Infinity : defaultInputProps;
```

Comment:
In languages other than English, this won't be `Margin` because in `margin.js` you're defining this as `__( 'Margin' )`, so it will be translated.
This should be `__( 'Margin' )` to take into account translations, though ideally I think it would be better to just use a bool `allowNegativeValues` (or something similar).

Additionally, `defaultInputProps` is defined as `{ min: 0 }` in lines 38-40, so perhaps the value here should be formatted similarly (`{ min: -Infinity }`)?

Reply:
Thanks, Ari.

> so perhaps the value here should be formatted similarly ({ min: -Infinity })

Why do we need this? 
What if we do instead `inputProps = allowNegativeValues ? -Infinity : defaultInputProps;`

Reply:
`inputProps` is an object, not a number. We use that below in line 111, where we do
```js
const inputControlProps = {
	...inputProps,
	onChange: handleOnChange,
	onFocus: handleOnFocus,
	isLinked,
	units,
	selectedUnits,
	setSelectedUnits,
	sides,
	values: inputValues,
};
```
If `inputProps` is a number and not an object (`{min:-Infinity}`), then what does it do in `inputControlProps`, and what is the minimum value?
Making it an object just makes it clear what happens where, and we keep things more consistent 👍